src/engine/SCons/Action.py

```diff
@@ -73,7 +73,7 @@
 """
 
 #
-# __COPYRIGHT__
+# Copyright (c) 2001, 2002, 2003, 2004, 2005, 2006, 2007 The SCons Foundation
```

Comment:
Somehow you've expanded these macros.  If you can use a simple text editor to undo these two changes, please fix this.